### ISSUE TITLE
Gulp stops running after some time (El Capitan upgrade)

### ISSUE THREAD

**Issue by shankiesan** [NicoSantangelo/sublime-gulp#52]
I'm having a wee problem with the Gulp package after upgrading to El Capitan.

On launching ST3, the Gulp package works as normal, but after some time, it stops working and new calls no longer bring up the list of tasks. No error message is displayed.

Any idea what might be going on?

Thanks!

**Comment by NicoSantangelo** [comment]
That's weird. I can't think of something that could break without throwing errors after the update, so two questions:

1. Running the same `gulp` on the terminal works?
2. What `gulp` tasks are you using?

Thanks!

**Comment by shankiesan** [comment]
I know! Very strange.

Answers:

1. `gulp` works fine on the terminal
2. I've got a handful of tasks running packages – the usual suspects such as bower, browser-sync, concat, uglify, compass, etc.

Thanks!

**Comment by NicoSantangelo** [comment]
I'm currently running El Capitan here so I'll give it a try and report back.
If you find a way to consistently reproduce the bug please let me know!

**Comment by shankiesan** [comment]
Good stuff. Indeed I'm hoping to find something consistent at the moment as well.

Can you think about how I'd go about completely removing the package? Is it just a matter of deleting the package's folder, or are there other bits kicking around that I need to be aware of? I was thinking that it might be worth starting from scratch.

**Comment by NicoSantangelo** [comment]
Good idea!

Deleting the folder should be enough. If you have custom user settings you could delete those too (`User` folder) but it's not really important. If you're using `Package Control` you can use `Package Control: Remove package`.
 Also, you could remove the `.sublime-gulp.cache` file of your project (you can use `Gulp: Delete Cache` if you want)

**Comment by shankiesan** [comment]
Hmm unfortunately that doesn't seem to have helped! I've removed and reinstalled and deleted the cache. Still can't find a pattern as to why it stops working.

There's another issue that I've been having since upgrading to El Capitan:

https://github.com/wbond/package_control/issues/1002#issuecomment-151703504

Do you think that could be related?

**Comment by NicoSantangelo** [comment]
I don't think so because is related with crypto and I'm using the basic `Thread` and `subprocess` Python libraries for `SublimeGulp` but I wouldn't discard it.

Sadly I couldn't reproduce the bug _yet_ and I have little free time lately because I'm behind schedule in some paying work but I'm slowly working on it (that said, if you find a specific way to replicate the bug, let me know!)

**Comment by shankiesan** [comment]
Ok, fair enough! I'm under it with client work at the moment as well! I'll let you know if I find a way of reproducing it.
